DISCLAIMER

```diff
@@ -1,11 +1,5 @@
-Apache EventMesh is an effort undergoing incubation at The Apache Software Foundation (ASF),
-sponsored by the Apache Incubator PMC.
-
-Incubation is required of all newly accepted projects until a further review indicates that the infrastructure, 
-communications, and decision-making process have stabilized in a manner consistent with other successful ASF projects.
-
-While incubation status is not necessarily a reflection of the completeness or stability of the code, 
-it does indicate that the project has yet to be fully endorsed by the ASF.
+Apache EventMesh, previously incubating at The Apache Software Foundation (ASF) 
+has successfully graduated from the incubator phase.
 
```

Comment:
You may take other apache project's DISCLAIMER file as a reference.

Reply:
I noticed that not many apache projects have DISCLAIMER file,

Reply:
A podling project must include a disclaimer in its repo, and a TLP doesn't have a disclaimer policy according to https://incubator.apache.org/policy/incubation.html#disclaimers. So you can just delete this file.

Reply:
The word DISCLAIMER also appears twice in the `.licenserc.yaml` file and can be removed.

Reply:
> A podling project must include a disclaimer in its repo, and a TLP doesn't have a disclaimer policy according to https://incubator.apache.org/policy/incubation.html#disclaimers. So you can just delete this file.

sure, thanks for you guidance. then i will delete this file